docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1900,6 +1900,86 @@ These are not currently modeled.
 #. Currently mailing addresses are the same as home addresses. This is not true, especially for rural populations. We plan to add this to the model later. 
 
 
+Social Security Observer
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Social security information is kept and tracked for a range 
+of different actions. For simplicity, we will limit this 
+section to only creation and dates of death. Others could be 
+added later, if desired. 
+
+**When to Sample** 
+
+- A sample of qualifying events is taken on every time step 
+
+**What to Sample** 
+
+.. list-table:: Simulant Attributes to Sample 
+  :widths: 20
+  :header-rows: 0
+
+  * - Unique simulant ID (for PRL tracking)
+  * - First name
+  * - Middle initial 
+  * - Last name
+  * - DOB 
+  * - Social Security Number 
+  * - Type of event 
+  * - Date of event 
+
+Currently, we will only track 2 types of events: 
+
+#. Creation of a SSN 
+#. Date of death recorded 
+
+The creation of a SSN can be triggered by a birth or by immigration into 
+the US. Both will be listed as "Creation" and are not differentiated. This 
+event is only recorded for simulates that receive a SSN. 
+The date of event is either the date of birth or the date of immigration. 
+If these values are unknown, any date within the time step can be recorded. 
+
+The date of death is a recording of anyone who has died and has a SSN. The 
+date of the event is the date of death. This will be listed as "Date 
+of Death". 
+
+Here is an example: 
+
+.. image:: SSA_example.png
+
+Note that the top row is a simulate that received a SSN at birth. The 
+third row indicates someone who immigrated at age 5 and received their 
+SSN then. 
+
+**Who to Sample** 
+
+100% of simulants with a SSN and a qualifying event in that time step 
+will be recorded. 
+
+**Data Errors/Noise** 
+
+- The omission rate (currently 0%) should be parameterizable. 
+- Additional noise in names and dates will be added in a separate function, at a later date. 
+
+**Limitations and Possible Future Adds** 
+
+There are a significant number of other possible "events" that 
+could be included in the observer. These are either not implemented 
+in the larger simulation at this time, or more difficult and so 
+have not been included. These are: 
+
+#. Name changes - either first names (trans folks or others) or last names (commonly marriage or divorce). Not included in larger simulation 
+#. Sex-coding changes - not included in larger simulation 
+#. Correction of incorrect information - not included in larger simulation 
```

Comment:
"Not included in larger simulation" doesn't seem like the right way to say this, since this event doesn't mean much outside the context of this observer.

By the way, this list of other events is a good example of something that I think would be infeasible or very difficult to do outside the sim (cannot be parameterizable), which is why it's especially important that we are clear that these are not happening in our Model 1 (see comment above).

Reply:
Okay, yeah, makes sense. I think this will be a tough add because to track a "correction" we first need to intentionally get something wrong and "remember" that error. I wrote a bit more to express this more fully in the note.